packages/ui-form-field/src/FormFieldLayout/index.tsx

```diff
@@ -154,7 +154,11 @@ class FormFieldLayout extends Component<FormFieldLayoutProps> {
         )
       }
       // needs to be wrapped because it needs an `id`
-      return <div id={this._labelId}>{this.props.label}</div>
+      return (
+        <div id={this._labelId} style={{ display: 'contents' }}>
```

Comment:
display:contents fixes the gap if the label is a ScreenreaderContent, this way it does not add a double margin